app/controllers/provider/admin/cms/builtin_legal_terms_controller.rb

```diff
@@ -58,6 +58,6 @@ def system_name_param
   end
 
   def permitted_params
-    params.permit(cms_template: %i[system_name draft])
+    params.permit(cms_template: %i[system_name draft published])
```

Comment:
Is adding `published` needed?

The controller always received the `draft`, its content is later copied to the `published` field on model save.

Reply:
Same as above, I pasted from the [model](https://github.com/3scale/porta/pull/3228/files#diff-3f006b9242fc9ffaa847d7677bbdea9d1b86adef6f462be03e80131417e4a46a). But yes, for this particular controller probably it's not needed, I'll check.